RNA-Seq/RNA-Seq-hands-on.md

```diff
@@ -40,32 +40,32 @@ This exercise uses RNA-seq data from the study by [Brooks et al. 2011](http://ge
 - Re-run **FastQC** and inspect the differences.
 
 
-**Step 2: Mapping of the reads with TopHat**
+**Step 2: Mapping of the reads with TopHat2**

```

Comment:
let's change all of them to TopHat then. In previous version, it was called TopHat2 sometimes.